Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -38,6 +38,27 @@ public IActionResult Library()
             return View(sortedList);
         }
 
+        public IActionResult Search()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult Search(NewSearch newSearch)
+        {
+            try
+            {
+                List<Book> bookList = SqlReference.Search(newSearch.Column, newSearch.Value);
+                newSearch.BookList = bookList;
+
+                return View(newSearch);
+            }
+            catch (Exception)
```

Comment:
At some point you might want to consider having this display and error in the same way that Andie did on her add book branch - feel free to make a ticket for that though and merge this now :)